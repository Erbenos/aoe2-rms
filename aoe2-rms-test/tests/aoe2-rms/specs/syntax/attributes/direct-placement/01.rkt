#lang aoe2-rms
; Directly place player 1 at the center of the map (50%, 50%)

<PLAYER-SETUP>
(direct-placement)

<LAND-GENERATION>
(create-land
 (terrain-type 'DESERT)
 (land-percent 3)
 (land-position 50 50)
 (assign-to-player 1))
