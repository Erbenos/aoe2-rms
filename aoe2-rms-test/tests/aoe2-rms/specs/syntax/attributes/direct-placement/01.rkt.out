<PLAYER_SETUP>
direct_placement
<LAND_GENERATION>
create_land {
  terrain_type DESERT
  land_percent 3
  land_position 50 50
  assign_to_player 1
}
